web/src/app/shared/components/projects/projects.component.html

```diff
@@ -5,6 +5,7 @@ <h3 mat-subheader>Projects</h3>
     <mat-icon mat-list-icon [matBadge]="project.repositories.length" matBadgeColor="accent" *ngIf="project.type === 'private'">lock</mat-icon>
     <h4 mat-line><a [routerLink]="['/projects', project.uid]">{{ project.title }}</a></h4>
     <p mat-line>{{ project.description }}</p>
+    <p mat-line *ngIf="project.url"><a href="{{project.url | stripeSlash}}">{{ project.url | stripeSlash }}</a></p>
```

Comment:
I think we should strip the slash on the way in not the way out

Reply:
If we will add a check while adding the url , then we have to check for the particular field then insert into database. If we have to add at multiple places then we have to do it same everywhere. We can format the URL anywhere in the application using just pipe operator.

Reply:
This is correct if we just want to display it differently. However, the service or model should be the central place to sanitise data, we should not save it incorrectly and then display it correctly 🤓

Reply:
okay I will fix it and remove the pipe.

Reply:
Thank you 👍